cmd/uhc/cluster/list/list.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright (c) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package list
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/openshift-online/uhc-cli/pkg/config"
+	"github.com/openshift-online/uhc-cli/pkg/util"
+	"github.com/openshift-online/uhc-sdk-go/pkg/client"
+	"github.com/openshift-online/uhc-sdk-go/pkg/client/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	debug     bool
+	parameter []string
+	managed   bool
+}
+
+var Cmd = &cobra.Command{
+	Use:   "list [MANAGED | UNMANAGED] ",
+	Short: "List clusters",
+	Long:  "List clusters. Defaults to managed.",
+	Run:   run,
+}
+
+func init() {
+	flags := Cmd.Flags()
+	flags.BoolVar(
```

Comment:
As mentioned above I think you will need to use `StringVar` and check explicitly if the string is empty, `true` or `false`.